scripts/aggregate-themes/theme-aggregator.js

```diff
@@ -130,22 +134,26 @@ class ThemeAggregator {
     }
 
     let themesToAggregate = defaultTheme ? [defaultTheme] : [];
-    themesToAggregate = themesToAggregate.concat(scoped);
+    if (scoped) {
+      themesToAggregate = themesToAggregate.concat(scoped);
+    }
 
+    const assets = [];
+    let asset;
     themesToAggregate.forEach((theme) => {
       asset = ThemeAggregator.aggregateTheme(theme, options, defaultFlag);
       if (asset) {
         if (asset.length > 1) {
           assets.push(...asset);
         } else {
-          assets.push(asset);
+          assets.push(asset.pop());
         }
       }
 
       if (defaultFlag) defaultFlag = false; // There can only be one instance of the default theme. This stops multiple root themes from being generated.
     });
 
-    return ThemeAggregator.writeJsFile(assets);
+    return assets;
```

Comment:
this function's description needs updated to reflect this change

Reply:
Should we return null if not files exist?

Reply:
we currently return an empty array - i'm not opinionated either way

Reply:
I would assume null would follow the pattern defined in aggregrateTheme: https://github.com/cerner/terra-toolkit/pull/337/files#diff-1672fd55492cdce050f5aa314b17029bR54